dotCMS/src/main/java/com/dotmarketing/webdav/DotWebdavHelper.java

```diff
@@ -1492,7 +1493,7 @@ private InputStream getResourceContent(String resourceUri, User user) throws Exc
                 Contentlet cont  = conAPI.findContentletByIdentifier(identifier.getId(), false, defaultLang, user, false);
 			    workingFile = cont.getBinary(FileAssetAPI.BINARY_FIELD);
 			}
-			FileInputStream is = new FileInputStream(workingFile);
+			InputStream is = Files.newInputStream(workingFile.toPath());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "workingFile" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2259)

Reply:
done